src/Web/E-Games.Web/E-Games.Web/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-E_Games.Web-53bc9b9d-9d6a-45d4-8429-2a2761773502;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=DESKTOP-58V9NB9\\SQLEXPRESS;Database=E-Games;Trusted_Connection=True;Encrypt=false;TrustServerCertificate=true;MultipleActiveResultSets=true"
```

Comment:
please change your desktop name to localhost, so that it would be easier for other devs to set up your project